src/common/tests_lib.c

```diff
@@ -51,7 +51,8 @@ dts_oid_gen(uint16_t oclass, uint8_t ofeats, unsigned seed)
 	/* generate a unique and not scary long object ID */
 	oid.lo	= obj_id_gen++;
 	oid.lo	|= hdr;
-	oid.hi	= rand() % 100;
+	/** Mod by large 32-bit prime */
+	oid.hi	= rand() % 2147483647;
```

Comment:
Oops.  I could revert this change.  On the other hand, it will make the value more unique.